ProcessLib/LiquidFlow/LiquidFlowLocalAssembler-imp.h

```diff
@@ -0,0 +1,114 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   LiquidFlowLocalAssembler.cpp
+ *
+ * Created on August 19, 2016, 2:28 PM
+ */
+
+#ifndef OGS_LIQUIDFLOWLOCALASSEMBLER_IMP_H
+#define OGS_LIQUIDFLOWLOCALASSEMBLER_IMP_H
```

Comment:
Until now everybody named the implementation files ending with '-impl.h'; don't break that please.